account_invoice_report_service/static/src/scss/report_invoice_delivery_styles.scss

```
@@ -45,7 +45,7 @@
     @include ris-basetable-mixin;
 }
 
-.rid-table2 > tbody > tr.rid-tr2-order-header > td {
+.border-bottom {
```

```suggestion
.rid-table2-border-bottom
```
And just a doubt, don't you need the `> td` like the other treport?
```suggestion
.rid-table2-border-bottom > td
```